python/xorbits/sklearn/cluster/__init__.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022-2023 XProbe Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from ...core.utils.fallback import unimplemented_func
+
+
+def _install():
+    """Nothing required for installing sklearn."""
+
+
+def __dir__():
+    try:  # pragma: no cover
```

Comment:
You could just add `# pragma: no cover` to the top-level func, for example, `def __dir__()` here.

Reply:
thanks, got it !